src/Container/NotFoundMiddlewareFactory.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2016-2017 Zend Technologies USA Inc. (https://www.zend.com)
```

Comment:
New file, 2018 only please.

Reply:
This is a _renamed_ file; it used to be `NotFoundHandlerFactory`, but the name changed due to correctly identifying `Zend\Expressive\Middleware\NotFoundHandler` as _middleware_.

Reply:
So as name is changed maybe we should also update year range then?